packages/data-loader/src/controllers/import.ts

```diff
@@ -39,10 +42,23 @@ export const buildImporter = ({
     const { app, filePath } = options;
     const buf = await fs.readFile(filePath);
     const type = extractFileType(filePath);
-    const records = parser(type, buf.toString());
+    const parser = buildParser(type);
+
+    const records =
+      type === "csv"
+        ? parser(buf.toString(), await fetchFieldsJson(options))
+        : // @ts-ignore
+          parser(buf.toString());
     await addAllRecordsChunk(app, records);
   }
 
+  async function fetchFieldsJson(options: Options) {
+    const fieldsJson: {
+      properties: Record<string, KintoneFormFieldProperty.OneOf>;
+    } = await apiClient.app.getFormFields(options);
```

Comment:
`getFormFields` can accept generics like this: 

```suggestion
    const fieldsJson = await apiClient.app.getFormFields<
      Record<string, KintoneFormFieldProperty.OneOf>
    >(options);
```

The result of the type `fieldsJson.properties` is the same, but I think it's more natural way to specify its type.